scripts/source-maintenance.sh

```diff
@@ -66,7 +139,10 @@ for FILENAME in `git ls-files`; do
 	#
 	# Code Style formatting maintenance
 	#
-        if test "${ASVER}"; then
+	for SCRIPT in `git ls-files scripts/maintenance/`; do
```

Comment:
Nit: why git ls-files and not just listing the files?

Reply:
This may not be a complete or even valid answer to your question, but:

* `git ls-files` ignores files ignored by git. This is very helpful when editors and such create helper/temporary files inside our directory.
* AFAICT, `git ls-files` has better (than `ls -R`) support for listing files in _sub_-directories. We might group scripts in sub-directories of the `scripts/maintenance` directory.

Reply:
Pretty much what Alex said plus the benefit that it ignores objects which are not in the repository. So unintentional files are not going to be arbitrarily executed.